### PR TITLE
fix: fail to route request to /author/id/posts

### PR DESCRIPTION
**Pull request by haotianzhu**
## Related User Stories

_[tag](https://help.github.com/en/articles/autolinked-references-and-urls) related stories or create a new one if does not exist_

## Check list

- [x] My branch is up to date with `master`. [How?](https://github.com/ExiaSR/hyperion/wiki/How-to-collaborate)
- [x] I passed all integration tests in Travis CI.
- [x] Now it is a good time to ask for review.

### CODE REVIEW

**Comment by haotianzhu** [comment]
[26/Mar/2019 04:32:46] "GET /author/1/posts HTTP/1.1" 200 697
[26/Mar/2019 04:32:52] "GET /author/1 HTTP/1.1" 200 504

**Comment by haotianzhu** [comment]
#145

File: api/urls.py [hunk 1]
```diff
@@ -58,10 +58,10 @@
         "posts/<int:pk>", post_views.PostViewSet.as_view({"get": "retrieve", "delete": "destroy"})
     ),
     path("posts/<int:pk>/comments", comment_views.CommentViewSet.as_view({"post": "new_comment"})),
-    path("author/<path:author_id>", user_views.get_profile, name="get_profile"),
     url(
```
Comment (ExiaSR) [inline api/urls.py]: @haotianzhu 要不干脆把这个也换成用`path`？
另外这个endpoint估计也要支持抓取foreign user的post

```py
path("author/<path:author_id>/posts", post_views.PostViewSet.as_view({"get": "get_author_id_posts"}))
```

Comment (ExiaSR) [reply api/urls.py]: 不过我们可以留到另外一个PR做，这个暂时先把author/id/posts用不了修了

Comment (haotianzhu) [reply api/urls.py]: 都可以 不过我的function args 是 pk
author/<path:author_id>/posts 不知道可以pass 过去么

Comment (ExiaSR) [reply api/urls.py]: @haotianzhu 这个在view里面就得加一点额外的handling了

可以参考这个，就是有点丑

https://github.com/ExiaSR/hyperion/blob/master/hyperion/views/user_views.py#L50

**Pull request merged**